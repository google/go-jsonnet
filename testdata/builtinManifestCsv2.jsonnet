std.manifestCsv([{ "head1": "val1" }, { "head1": "val2" }])