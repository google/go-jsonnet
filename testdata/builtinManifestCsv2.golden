"head1\nval1\nval2\n"
